lib/ansible/modules/cloud/azure/azure_rm_virtualmachine_scaleset_facts.py

```diff
@@ -187,6 +205,53 @@ def exec_module(self, **kwargs):
         else:
             self.results['ansible_facts']['azure_vmss'] = self.list_items()
 
+        if self.format == 'ansible':
+            for index in range(len(self.results['ansible_facts']['azure_vmss'])):
+                vmss = self.results['ansible_facts']['azure_vmss'][index]
+                subnet_id = vmss['properties']['virtualMachineProfile']['networkProfile']['networkInterfaceConfigurations'][0]['properties']['ipConfigurations'][0]['properties']['subnet']['id']
+                backend_address_pool_id = vmss['properties']['virtualMachineProfile']['networkProfile']['networkInterfaceConfigurations'][0]['properties']['ipConfigurations'][0]['properties']['loadBalancerBackendAddressPools'][0]['id']
+                subnet_name = re.sub('.*subnets\\/', '', subnet_id)
+                load_balancer_name =  re.sub('\\/backendAddressPools.*', '', re.sub('.*loadBalancers\\/', '', backend_address_pool_id))
+                virtual_network_name = re.sub('.*virtualNetworks\\/', '', re.sub('\\/subnets.*', '', subnet_id))
+
+                updated = {}
```

Comment:
is it possible to group those updatable fields in some constant list?

Reply:
yes, i think so